pywps/inout/basic.py

```diff
@@ -5,6 +5,7 @@
 
 from pywps._compat import text_type, StringIO
 import os
+from io import open
```

Comment:
maybe use it in the form:
```
import io
io.open()
```

and add a comment like "PY2 compatibility". When we skip PY2 we can simplify it again.

Reply:
see here:
https://stackoverflow.com/questions/40809901/python-2-7-io-open-use-from-io-import-open-or-io-open#40809945

Reply:
... I will add this in a follow-up PR.